File: ksb-front/src/app/shared/dts/agence-dto.ts
import { EmployeeDto } from 'src/app/shared/dts/employee-dto';

export class AgenceDto {
    id!: string;
    nom!: string;
    adresse!: string;
    region!: string;
    departement!: string;
    telephone!: string;
    dateCreation!: string;
    employees!: EmployeeDto[];
}

File: ksb-immo/src/main/java/sn/ksb/immo/ksbimmo/application/dtos/AgenceDto.java
package sn.ksb.immo.ksbimmo.application.dtos;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AgenceDto {
    private String id;
    private String nom;
    private String region;
    private String departement;
    private String telephone;
    private String adresse;

    private String dateCreation;
    private List<EmployeeDto> employees;
}
